civis/cli/_cli_commands.py

```diff
@@ -156,6 +165,55 @@ def _str_table_result(cols, rows):
     return '\n'.join(tb_strs)
 
 
+@click.command(
+    'follow-log',
+    help='Output live log from the most recent job run.' + _FOLLOW_LOG_NOTE)
+@click.argument('id', type=int)
+def jobs_follow_log(id):
+    client = civis.APIClient()
+    runs = client.jobs.list_runs(id, limit=1, order='id', order_dir='desc')
+    if not runs:
+        raise click.ClickException('No runs found for that job ID.')
+    run_id = runs[0].id
+    print('Run ID: ' + str(run_id))
+    _jobs_follow_run_log(id, run_id)
+
+
+@click.command(
+    'follow-run-log',
+    help='Output live run log.' + _FOLLOW_LOG_NOTE)
+@click.argument('id', type=int)
+@click.argument('run_id', type=int)
+def jobs_follow_run_log(id, run_id):
+    _jobs_follow_run_log(id, run_id)
+
+
+def _jobs_follow_run_log(id, run_id):
+    client = civis.APIClient(return_type='raw')
+    seen_max_log_id = 0
+    continue_polling = True
+
+    while continue_polling:
+        response = client.jobs.list_runs_logs(id, run_id,
+                                              last_id=seen_max_log_id)
+        if 'civis-max-id' in response.headers:
```

Comment:
It's the maximum log id that has been seen on Platform. It's used to decide if we need to keep retrieving logs instead of waiting for the next 3 second poll interval.
Maybe I could rename the variable here to `max_log_id_seen_on_platform`?

Reply:
Part of the issue is that I don't know how platform stores log information. Are you saying that each log line has its own id?

Currently the people who maintain the API clients are not people who work on the API itself, so an inline comment mentioning that we can check for new platform logs by looking for new ids would help clarify.

Reply:
Ah, okay. Yeah, each log message (I'm not calling it a line because I think a single message could technically have multiple newlines) has an ID that should be unique within each run and is in chronological order based on when the message was originally sent. 
Here's what one of the log entries from `civis jobs list-runs-logs` looks like in YAML format, since that's the same endpoint gets used here:
```
- createdAt: '2019-11-04T15:56:44.492Z'
  id: 492602116
  level: info
  message: Queued
```